Log message:
MAX_NR_ZONES changed, so use correct defines now.

Modified files:
    linux/arch/m68k/mm:
	motorola.c sun3mmu.c 


Index: linux/arch/m68k/mm/motorola.c
Stats: 12 modifications
http://linux-m68k-cvs.ubb.ca/c/cvsweb/linux/arch/m68k/mm/motorola%2ec.diff?r1=1.12&r2=1.13
========================================================================
--- motorola.c	4 Aug 2004 21:02:05 -0000	1.12
+++ motorola.c	28 Jan 2006 21:29:07 -0000	1.13
@@ -203,7 +203,7 @@
 {
 	int chunk;
 	unsigned long mem_avail = 0;
-	unsigned long zones_size[3] = { 0, };
+	unsigned long zones_size[MAX_NR_ZONES] = { 0, };
 
 #ifdef DEBUG
 	{
@@ -257,12 +257,12 @@
 #ifdef DEBUG
 	printk ("before free_area_init\n");
 #endif
-	zones_size[0] = (mach_max_dma_address < (unsigned long)high_memory ?
-			 (mach_max_dma_address+1) : (unsigned long)high_memory);
-	zones_size[1] = (unsigned long)high_memory - zones_size[0];
+	zones_size[ZONE_DMA] = (mach_max_dma_address < (unsigned long)high_memory ?
+				(mach_max_dma_address+1) : (unsigned long)high_memory);
+	zones_size[ZONE_NORMAL] = (unsigned long)high_memory - zones_size[0];
 
-	zones_size[0] = (zones_size[0] - PAGE_OFFSET) >> PAGE_SHIFT;
-	zones_size[1] >>= PAGE_SHIFT;
+	zones_size[ZONE_DMA] = (zones_size[ZONE_DMA] - PAGE_OFFSET) >> PAGE_SHIFT;
+	zones_size[ZONE_NORMAL] >>= PAGE_SHIFT;
 
 	free_area_init(zones_size);
 }
Index: linux/arch/m68k/mm/sun3mmu.c
Stats: 5 modifications
http://linux-m68k-cvs.ubb.ca/c/cvsweb/linux/arch/m68k/mm/sun3mmu%2ec.diff?r1=1.8&r2=1.9
========================================================================
--- sun3mmu.c	17 Oct 2004 18:16:59 -0000	1.8
+++ sun3mmu.c	28 Jan 2006 21:29:07 -0000	1.9
@@ -46,7 +46,7 @@
 	unsigned long address;
 	unsigned long next_pgtable;
 	unsigned long bootmem_end;
-	unsigned long zones_size[3] = {0, 0, 0};
+	unsigned long zones_size[MAX_NR_ZONES] = { 0, };
 	unsigned long size;
 
 
@@ -92,8 +92,7 @@
 	current->mm = NULL;
 
 	/* memory sizing is a hack stolen from motorola.c..  hope it works for us */
-	zones_size[0] = ((unsigned long)high_memory - PAGE_OFFSET) >> PAGE_SHIFT;
-	zones_size[1] = 0;
+	zones_size[ZONE_DMA] = ((unsigned long)high_memory - PAGE_OFFSET) >> PAGE_SHIFT;
 
 	free_area_init(zones_size);
 
-
To unsubscribe from this list: send the line "unsubscribe linux-m68k-cvscommit" in
the body of a message to majordomo@vger.kernel.org
More majordomo info at  http://vger.kernel.org/majordomo-info.html

